models/Program.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * custom type enumerating days of the week for the createProgramInput custom type
+ */
```

Comment:
for enums like `provinces` and `roles`, I imported them directly from `@prisma/client` so that what's stored in the database is always consistent with what we're using in our codebase. What are your thoughts on that approach v.s. defining it here, since we should probably be consistent? @feng-eric @mahadzaryab1

Reply:
Yes, I think that's definitely better.